modules/appeals_api/spec/services/appeals_api/pdf_construction/generator/generator_hlr_spec.rb

```diff
@@ -153,6 +153,20 @@
       #     }
       #   }
       # end
+
+      # These specs need to be skipped until we can determine why they fail in the k8s branch but not master:
```

Comment:
The `k8s` branch is no longer used for vets-api (we deploy from `master` now) so this comment can go 😄 

Also, the test right above this one ☝️, can it be uncommented out now?

Reply:
The plan is to re-evaluate these tests after this PR get merged....in the past these test have been fragile, sometimes passing, sometimes not, and we don't want to conflate the new version of the form changes with these failing test.  We do run them locally when we change the from generation code.

(We think the issue with these test is slightly different behavior of the PDFtk library on the k8 linux vs our OS X development machines)